osu.Game/Screens/Edit/Compose/Components/SelectionBox.cs

```diff
@@ -300,7 +301,7 @@ private void addYFlipComponents()
             addButton(FontAwesome.Solid.ArrowsAltV, "Flip vertically", () => OnFlip?.Invoke(Direction.Vertical, false));
         }
 
-        private void addButton(IconUsage icon, string tooltip, Action action)
+        private SelectionBoxButton addButton(IconUsage icon, string tooltip, Action action)
```

Comment:
Rather than returning the button and storing to fields, we could probably move the hotkey handling to inside the button. Maybe something for the future (although if it's seen as a better direction I will apply it here).

Reply:
I actually prefer this more than handling the key in the buttons themselves because I've tried handling hotkeys in buttons before on the mod overlay and it failed miserably as soon as the first hotkey conflict happened. So centralised is _probably_ better in this case.

Reply:
Also see https://github.com/ppy/osu/pull/24059/commits/9be2d9d62e131d22e98b650bdb82e48c97d8f18a - fixing that in the buttons would require further rejiggling.